src/Configuration.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  Configuration.swift
+//  Sango
+//
+//  Created by Adam Shea on 5/7/18.
+//  Copyright © 2018 Afero, Inc. All rights reserved.
+//
+
+import Foundation
+
+public struct Configuration {
+    
+    var inputAssets:String
+    var inputs:[String]
+    var platform:Platform
+    var identifierFilename:String?
+    var outLocation:String
+    var package:String?
+    
+    init(inputAssets:String, inputs:[String], platform:Platform,
+         identifierFilename:String?, outLocation:String, package:String?) {
+        self.inputAssets = inputAssets
+        self.inputs = inputs
+        self.platform = platform
+        self.identifierFilename = identifierFilename
+        self.outLocation = outLocation
+        self.package = package
+    }
+
+    public static func fromConfigFile(text:[String:Any]) -> Configuration {
+        let inputAssets = text["input_assets"] as? String
+        let inputFiles = text["inputs"] as? [String]
+        let identifierFilename = text["out_identifier_name"] as? String
+        let outLocation = text["out_location"] as? String
+        let platform = getPlatformFromString(platformString: text["platform"] as? String)
+        let package = text["package"] as? String
+        
+        if (outLocation == nil) {
```

Comment:
no need for parenthesis on your 'if' statements
if outLocation == nil {   is generally enough

Reply:
Oh! I see what we're doing here.

```
guard let outLocation = outLocation else {
    //error
    exit(-1)
}
```
Then down below, you don't have to force unwrap your optional with "!"
Feel free to grab me for the standard ideas behind unwrapping optionals.